### PR TITLE
895 add issue templates

### LINKED ISSUE #895: Add Issue Templates
Try click "new issue" button in https://github.com/bitshares/bitshares-ui.

It's done with https://github.com/bitshares/bitshares-ui/tree/staging/.github/ISSUE_TEMPLATE

More info:
https://help.github.com/articles/setting-up-your-project-for-healthy-contributions/

### PR DESCRIPTION
**Pull request by ryanRfox**
Resolves #895 by adding:
* New folder structure: /.github/ISSUE_TEMPLATE
* New template files for:
  * Bug Fix
  * Feature Enhancement
Result will be when a GitHub user clicks the `New issue` button they will choose between the two templates. The Core Team will evaluate and prioritize the Issue thereafter by modifying the Title, Description, Lables, etc.

### CODE REVIEW

**Comment by jmjatlanta** [comment]
Many unrelated commits. Perhaps rebase (may require a new PR)?

**Comment by clockworkgr** [comment]
@jmjatlanta was just about to post that

**Comment by abitmore** [comment]
I guess we need to do this based on `master` branch, like it's a hotfix?

**Comment by abitmore** [comment]
Bug report should not only be about `witness_node`, for example, can be a `cli_wallet` bug. So we also need to know the "scope" or "affected components" or something similar.

**Comment by pmconrad** [comment]
Changed PR to target master. According to github documentation, the files must be on the "default branch" (that's master for us).

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain process flow and behavior.
+
+**witness_node --version (please complete the following information):**
+ - Version: [e.g. 2.0.180425]
+ - SHA: [e.g. bf263b1db47a91ea9748118df1abb5a22cafc40b]
+ - SSL: [e.g. OpenSSL 1.0.2g  1 Mar 2016]
+ - Boost: [e.g. 1.62]
+ - Websocket++: [e.g. 0.7.0]
```
Comment (pmconrad) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Websocketpp is a submodule in our sources, system library version doesn't matter.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+**Expected Behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain process flow and behavior.
+
+**witness_node --version (please complete the following information):**
+ - Version: [e.g. 2.0.180425]
+ - SHA: [e.g. bf263b1db47a91ea9748118df1abb5a22cafc40b]
```
Comment (pmconrad) [inline .github/ISSUE_TEMPLATE/bug_report.md]: SHA is redundant, perhaps combine with previous line to "Version/Git Tag/Commit-ID"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* witness_node --version (optional)
+* Aditional Context (optional)
+
+Finally, press the 'Submit new issue' button. The Core Team will evaluate and prioritize your Bug Report for future development. 
+
+**Bug Description**
+A clear and concise description of what the bug is.
+
+**Steps To Reproduce**
+Steps to reproduce the behavior:
```
Comment (pmconrad) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Please make it clear that the following steps are an example only.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+**Instructions**
+Pleaae include a detailed Title above. Next, please complete the following sections below:
+* Bug Description
+* Steps To Reproduce
+* Expected Behavior
+* Screenshots (optional)
+* witness_node --version (optional)
+* Aditional Context (optional)
```
Comment (pmconrad) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Typo: Ad*d*itional

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+---
+name: Bug Report
+about: Create a detailed report about a deficiency in the BitShares Core implementation.
+
+---
+
+**Instructions**
+Pleaae include a detailed Title above. Next, please complete the following sections below:
```
Comment (pmconrad) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Typo: Plea*s*e

**Comment by ryanRfox** [comment]
@pmconrad I have made the requested changes. Note: the `Impacts` section of the Feature Request document are intended to match the `Labels` we assign to Issues. I agree, 'Consensus' may not be the best term. I renamed it to 'Protocol' to match the conversation the Core Team discussed related to 'Protocol Upgrade' rather than 'Consensus Impacting' going forward.

**Comment by ryanRfox** [comment]
This PR included a file which it should not have. Closing and will create a new proper PR.

**Pull request closed without merging**